src/quacc/calculators/mrcc/io.py

```diff
@@ -0,0 +1,126 @@
+from __future__ import annotations
+
+from io import StringIO
+from pathlib import Path
+
+from ase.io import read
+from ase.units import Hartree
+from ase.utils import reader, writer
+
+# Made from NWChem interface
+
+
+@reader
+def read_geom_mrccinp(fd):
+    """Method to read geometry from an MRCC input file."""
+    lines = fd.readlines()
+    atomslength = 0
+    atomslenline = -1
```

Comment:
Why are we initializing `atomslength` to 0? Presumably, the length must be nonzero, right?  Assuming that's the case, the better thing to do here would not be to initialize with 0 but rather to raise an error if the geometry info isn't parsed.

Also, let's use standard convention of underscores in variable names, e.g. `atoms_length`.